h/streamer/__init__.py

```diff
@@ -1,4 +1,6 @@
-# -*- coding: utf-8 -*-
+from h.streamer.tweens import close_db_session_tween_factory
+
+__all__ = ["close_db_session_tween_factory"]
```

Comment:
Using `__all__` to explicitly declare the public interface of a package, as per PEP 8.

Also be importing `close_db_session_tween_factory` here and putting into `__all__` it becomes `h.streamer.close_db_session_tween_factory` not `h.streamer.tweens.close_db_session_tween_factory`. So the structure of the package namespace is flattened from the point of view of its public interface. This means you can reorganise things inside the package, without affecting any user code outside of the package.

Reply:
This is another thing that isn't getting done, but good practice non-the-less.